clang-tools-extra/clang-tidy/misc/IncludeCleanerCheck.cpp

```diff
@@ -209,14 +212,16 @@ void IncludeCleanerCheck::check(const MatchFinder::MatchResult &Result) {
     // main file.
     if (auto Replacement =
             HeaderIncludes.insert(llvm::StringRef{Spelling}.trim("\"<>"),
```

Comment:
insert -> emplace

Reply:
`HeaderIncludes` is NOT a container. This `insert` means to create a `tooling::Replacement` object rather than container insertion/emplacing.